Resources/Textures/Objects/Specific/Research/anomalyscanner.rsi/meta.json

```
@@ -4,11 +4,19 @@
     "x": 32,
     "y": 32
   },
-  "license": "CC0-1.0",
-  "copyright": "Created by EmoGarbage",
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Created by EmoGarbage, Inhands by TiniestShark based off of /tg station's health analyzer at https://github.com/tgstation/tgstation/commit/c7e5ca401c91cb1351c2f68cda3fe35634c4dba2",
   "states": [
     {
       "name": "icon"
+    },
+    {
+        "name": "inhand-left",
+        "directions": 4
+    },
+    {
+        "name": "inhand-right",
+        "directions": 4
```

After discussing with Slartibartfast in the Discord, they suggested just making sure you are still aligned with the way the file originally did it.

Unfortunately, I don't think there's a better way of dealing with this in your editor outside of just knowing to indent by 2 spaces.

```suggestion
      "name": "inhand-left",
      "directions": 4
    },
    {
      "name": "inhand-right",
      "directions": 4
```

---

> yeah, it's a little messy
> I think the most common one is 2 spaces, because that is what RSIedit gives you when you autoconvert a .dmi to a rsi
> so for new meta files I would do that, for old ones I would keep the same intendation as before in order not to mess with the git history"
- https://discord.com/channels/310555209753690112/310555209753690112/1316957226446880778

---

Ah all right, I'll keep that in mind for the future then, appreciated! I was hoping to piece together how to do it though deconstructing other PRs but I think I should just force my adhd ass to sit through the proper tutorials for the next one I want to do.